pycross/private/tools/pdm_translator.py

```diff
@@ -58,6 +58,11 @@ def get_development_dependencies(lock: Dict[str, Any]) -> Dict[str, List[Require
     return {group: [Requirement(EDITABLE_PATTERN.sub("", dep)) for dep in deps] for group, deps in dep_groups.items()}
 
 
+def get_excluded_packages(lock: Dict[str, Any]) -> List[str]:
+    packages = lock.get("tool", {}).get("pdm", {}).get("resolution", {}).get("excludes", [])
```

Comment:
What is this section? Do you have an example lock file?

Reply:
we make use of [tool.pdm.resolution.excludes](https://pdm-project.org/latest/usage/config/#exclude-specific-packages-and-their-dependencies-from-the-lock-file).
packages listed there will also not be contained in the lockfile, that's why i explicitly filter them out here.
I'm not completely sure if this is required or not.

Reply:
I see. So this seems orthogonal to the multi-platform issue from #115, right? If that's the case, I don't have a problem with this change. It seems like we'll need to handle both cases.

Do you know why the tests are failing?